Content.Server/Procedural/DungeonSystem.Commands.cs

```diff
@@ -44,7 +42,8 @@ private async void GenerateDungeon(IConsoleShell shell, string argstr, string[]
         }
 
         var position = new Vector2i(posX, posY);
-        var dungeonUid = _mapManager.GetMapEntityId(mapId);
+        _maps.TryGetMap(mapId, out var dungeonUidNullable);
+        var dungeonUid = dungeonUidNullable!.Value;
```

Comment:
You're using a Try* version of the fetch, but then ignoring the success status and assuming it succeeded. What you should instead be doing is one of these:

* Use `GetMap()` directly if you're certain the map exists.
* Use `GetMapOrInvalid()` to preserve the existing behavior.

Do the latter, as I don't want to have warning fix PRs like this possibly changing behavior.

Reply:
Yeah, I thought about `GetMapOrInvalid` too, but wasn't sure if I should use it. Will fix it